tests/unit/test_eel.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import eel
+import pytest
+from tests.utils import TEST_DATA_DIR
+
+INIT_DIR = TEST_DATA_DIR / 'init_test'
+"""Directory for testing eel.__init__."""
+
+
+@pytest.mark.parametrize('js_code, expected_matches', [
+    ('eel.expose(w,"say_hello_js")', ['"say_hello_js"']),
+    ('eel.expose(function(e){console.log(e)},"show_log_alt")', ['"show_log_alt"']),
+    (' \t\nwindow.eel.expose((function show_log(e) {console.log(e)}), "show_log")\n', ['"show_log"']),
+    ((INIT_DIR / 'minified.js').read_text(), ['"say_hello_js"', '"show_log_alt"', '"show_log"']),
+    ((INIT_DIR / 'sample.html').read_text(), ['say_hello_js']),
```

Comment:
Could we possibly extend this to also check the remaining two files in that dir - App.tsx and hello.html?